examples/echo.roc

```diff
@@ -1,21 +1,25 @@
-app [Model, server] { pf: platform "../platform/main.roc" }
+app [Model, init!, respond!] { pf: platform "../platform/main.roc" }
 
 import pf.Stdout
 import pf.Http exposing [Request, Response]
 import pf.Utc
 
 Model : {}
 
-server = { init: Task.ok {}, respond }
+init! : {} => Result Model []
+init! = \{} -> Ok {}
 
-respond : Request, Model -> Task Response [ServerErr Str]_
-respond = \req, _ ->
+respond! : Request, Model => Result Response [StdoutErr _]
+respond! = \req, _ ->
     # Log request datetime, method and url
-    datetime = Utc.now! |> Utc.toIso8601Str
-    Stdout.line! "$(datetime) $(Http.methodToStr req.method) $(req.url)"
+    datetime = Utc.to_iso_8601 (Utc.now! {})
+
+    Stdout.line!? "$(datetime) $(Inspect.toStr req.method) $(req.uri)"
```

Comment:
I'll make a single comment to capture the general thought: We _probably_ want to use `try` as per Richard's ask until parens are ready, but ? is more future-proof since we'll switch back once we get there. I'm really not sure which one we should use, maybe using both is good to show you can use either? 🤷🏼

Reply:
Good pickup, I've been trying to use `try` exclusively in the examples. I'll update

Reply:
Done in bf53228